test/e2e/README.md

```diff
@@ -40,10 +40,16 @@ You can install all the dependencies with [brew](https://brew.sh/):
 If an action is required on your part, `brew` will let you know in its log
 messages.
 
+##### Fork and Clone Antrea
+* Fork https://github.com/vmware-tanzu/antrea on github
+* mkdir -p antrea/github.com/antrea; cd $_
+* git clone https://github.com/<your-name>/antrea
```

Comment:
Hi @Dhruv-J - Thanks for your PR. However, this info is already included in https://github.com/vmware-tanzu/antrea/blob/master/CONTRIBUTING.md, which is targeted at new contributors. This README is itself is orthogonal to that: you don't need to fork the repo to build & test Antrea.